include/grpcpp/security/credentials_impl.h

```diff
@@ -129,6 +129,14 @@ class CallCredentials : private grpc::GrpcLibraryCodegen {
   virtual SecureCallCredentials* AsSecureCredentials() = 0;
 };
 
+struct TlsVersion {
+  TlsVersion() : min_tls_version(TLS1_2), max_tls_version(TLS1_3) {}
+  TlsVersion(grpc_tls_version min_tls_version, grpc_tls_version max_tls_version)
```

Comment:
I think the arguments and the fields should have different names. For example, the arguments could be called `min_version` and `max_version`.